internal/slack/resource_slack_alert.go

```diff
@@ -99,6 +100,12 @@ func (r *SlackAlertResource) Schema(ctx context.Context, req resource.SchemaRequ
 				Computed:            true,
 				Default:             booldefault.StaticBool(false),
 			},
+			"send_direct_message": schema.BoolAttribute{
```

Comment:
I think this needs to be updated
```suggestion
			"send_direct_message_to_approver": schema.BoolAttribute{
```